bootstrap.sh

```diff
@@ -131,6 +140,9 @@ action_baseline() {
     sudo apt install -y just
   fi
   pipx install gh-release-install
+  if [[ -n "$WSL_DISTRO_NAME" && "$(distro_name)" == "debian" ]]; then
```

Comment:
```suggestion
  if [[ -n "$WSL_DISTRO_NAME" ]]; then
```

Because wslu is "available" in ubuntu as a standard package; then we can just try and install it
- If it's debian proper; then we will have installed the repo
- If it's ubuntu then it's available.

I primarily use gh in token mode, so I didn't hit this problem, but I should have. Having `wslu` installed as a baseline package is appropriate (it wasn't installed on my wsl+ubuntu instance, and now I have pulseaudio installed which should make me sadder).